llvm/docs/LangRef.rst

```diff
@@ -9674,7 +9678,8 @@ This instruction requires several arguments:
    indicates the function accepts a variable number of arguments, the
    extra arguments can be specified.
 #. '``fallthrough label``': the label reached when the inline assembly's
-   execution exits the bottom.
+   execution exits the bottom. In case of an intrinsic call, the semantic
+   depends on the semantic of the intrinsic.
```

Comment:
This needs rephrasing, "the semantic depends on the semantic" doesn't read well. Does this really need changing, the control flow logically still goes to the fall through